Zilon.Core/Zilon.Core/MapGenerators/StaticObstaclesGenerator.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Analyzer 'SonarAnalyzer.Rules.CSharp.NullPointerDereference' threw an exception of type 'SonarAnalyzer.SymbolicExecution.SymbolicExecutionException' with message 'Error processing method: CreateAsync ## Method file: StaticObstaclesGenerator.cs ## Method line: 23,8 ## Inner exception: System.NullReferenceException: Object reference not set to an instance of an object ##   at SonarAnalyzer.SymbolicExecution.CSharpExplodedGraph.VisitDefaultExpression (Microsoft.CodeAnalysis.SyntaxNode instruction, SonarAnalyze](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5361554)